.github/workflows/event-merge-to-queue.yml

```diff
@@ -15,87 +15,69 @@ concurrency:
   get-latest-redis-tag:
     uses: ./.github/workflows/task-get-latest-tag.yml
     with:
       repo: redis/redis
-      prefix: 6 # Only tags starting with 6.x.x will be considered.
+  get-latest-7-0-redis-tag:
+    uses: ./.github/workflows/task-get-latest-tag.yml
+    with:
+      repo: redis/redis
+      prefix: 7.0
+  get-latest-6-2-redis-tag:
+    uses: ./.github/workflows/task-get-latest-tag.yml
+    with:
+      repo: redis/redis
+      prefix: 6.2
+  get-latest-6-0-redis-tag:
+    uses: ./.github/workflows/task-get-latest-tag.yml
+    with:
+      repo: redis/redis
+      prefix: 6.0
 
   docs-only: # Check whether the PR is only modifying docs
     uses: ./.github/workflows/task-check-docs.yml
 
-  test-linux:
-    needs: [print-configurations, docs-only, get-latest-redis-tag]
+  test-linux-x86_64:
+    needs: [docs-only, get-latest-redis-tag, get-latest-7-0-redis-tag, get-latest-6-2-redis-tag, get-latest-6-0-redis-tag]
     if: needs.docs-only.outputs.only-docs-changed == 'false'
     strategy:
       matrix:
-        test-config: # configurations for both standalone and coordinator
-          ${{ fromJson(needs.print-configurations.outputs.test-configurations) }}
-        include: # spacial configurations
-          ${{ fromJson(needs.print-configurations.outputs.include-configurations) }}
+        redis-ref:
+          - ${{ needs.get-latest-redis-tag.outputs.tag }}
+          - ${{ needs.get-latest-7-0-redis-tag.outputs.tag }}
+          - ${{ needs.get-latest-6-2-redis-tag.outputs.tag }}
+          - ${{ needs.get-latest-6-0-redis-tag.outputs.tag }}
+    uses: ./.github/workflows/flow-linux-platforms.yml
+    secrets: inherit
+    with:
+      redis-ref: ${{ matrix.redis-ref }}
+      architecture: x86_64
 
+  test-linux-aarch64:
+    needs: [docs-only, get-latest-redis-tag]
+    if: needs.docs-only.outputs.only-docs-changed == 'false'
     uses: ./.github/workflows/flow-linux-platforms.yml
     secrets: inherit
     with:
       redis-ref: ${{ needs.get-latest-redis-tag.outputs.tag }}
-      test-config: CONFIG=${{ matrix.test-config }}
-      standalone:  ${{ matrix.standalone  != 'false' }} # Set the default value to `true` if standalone is absent
-      coordinator: ${{ matrix.coordinator != 'false' }} # Set the default value to `true` if coordinator is absent
+      test-config: QUICK=1
+      architecture: aarch64
 
   test-macos:
-    needs: [print-configurations, docs-only, get-latest-redis-tag]
+    needs: [docs-only, get-latest-redis-tag]
     if: needs.docs-only.outputs.only-docs-changed == 'false'
     strategy:
-      fail-fast: false
       matrix:
-        dividor: [true, false]
-        # configurations for both standalone and coordinator
-        test-config: ${{ fromJson(needs.print-configurations.outputs.test-configurations) }}
-        # spacial configurations
-        include: ${{ fromJson(needs.print-configurations.outputs.include-configurations) }}
-
+        mode: [standalone, coordinator]
```

Comment:
I renamed the "splitting" parameter so it will be more clear what is the configuration for each job (from `test-macos (false)` and `test-macos (true)` to `test-macos (coordinator)` and `test-macos (standalone)`)